### PR TITLE
DE Keyboard Merge 

### PR DESCRIPTION
**Pull request by nyfz18**
### Contributor checklist

- [#33 ] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Related issue

- #33 

Added expanded DE keyboard settings

### CODE REVIEW

File: Keyboards/LanguageKeyboards/German/DEInterfaceVariables.swift [hunk 1]
```diff
@@ -51,6 +51,24 @@ public enum GermanKeyboardConstants {
     ["selectKeyboard", "ABC", "space", "ABC", "hideKeyboard"], // "undoArrow"
   ]
 
+  //added for expanded keyboard
+  static let letterKeysPadExpanded = [
+    ["^", "1", "2", "3", "4", "5", "6", "7", "8", "9", "0", "ß", "'", "delete"],
+    ["indent", "q", "w", "e", "r", "t", "z", "u", "i", "o", "p", "ü", "+", "return"],
+    ["uppercase", "a", "s", "d", "f", "g", "h", "j", "k", "l", "ö", "ä", "#", "return"],
+    ["shift", "<", "y", "x", "c", "v", "b", "n", "m", ",", ".", "-", "shift"],
+    ["selectKeyboard", ".?123", "microphone", "space", ".?123", "scribble", "hideKeyboard"], // "undoArrow"
```
Comment (andrewtavis) [inline Keyboards/LanguageKeyboards/German/DEInterfaceVariables.swift:60]: You can remove microphone from the row here and add it to the comments with `"undoArrow"` :)

Comment (andrewtavis) [reply Keyboards/LanguageKeyboards/German/DEInterfaceVariables.swift:60]: And feel free to do the same with `"scribble"` :)

File: Keyboards/LanguageKeyboards/German/DEInterfaceVariables.swift [hunk 2]
```diff
@@ -51,6 +51,24 @@ public enum GermanKeyboardConstants {
...
+  ]
+
+  //added for symbolKeysPadExpanded 
+  static let symbolKeysPadExpanded = [ 
+    ["'", "1", "2", "3", "4", "5", "6", "7", "8", "9", "0", "<", ">", "delete"],
```
Comment (andrewtavis) [inline Keyboards/LanguageKeyboards/German/DEInterfaceVariables.swift:65]: The first character here is a backtick (`` ` ``), right?

File: Keyboards/LanguageKeyboards/German/DEInterfaceVariables.swift [hunk 3]
```diff
@@ -51,6 +51,24 @@ public enum GermanKeyboardConstants {
...
+    ["indent", "[", "]", "{", "}", "#", "%", "^", "*", "+", "=", "\"", "|", "return"], 
+    ["undo", "-", "/", ":", ";", "(", ")", "$", "&", "@", "£", "¥", "~", "return"], 
+    ["redo", " ", "...", ".", ",", "?", "!", "'", '"', "_", "€"],
```
Comment (andrewtavis) [inline Keyboards/LanguageKeyboards/German/DEInterfaceVariables.swift:68]: It looks like there's a problem here with the characters. The first `'` in `'"'` seems to be not connected to the second. Is it maybe a different quote character than the second?

File: Keyboards/LanguageKeyboards/German/DEInterfaceVariables.swift [hunk 4]
```diff
@@ -51,6 +51,24 @@ public enum GermanKeyboardConstants {
...
+    ["selectKeyboard", ".?123", "microphone", "space", ".?123", "scribble", "hideKeyboard"], // "undoArrow"
+  ]
+
+  //added for symbolKeysPadExpanded 
+  static let symbolKeysPadExpanded = [ 
+    ["'", "1", "2", "3", "4", "5", "6", "7", "8", "9", "0", "<", ">", "delete"],
+    ["indent", "[", "]", "{", "}", "#", "%", "^", "*", "+", "=", "\"", "|", "return"], 
+    ["undo", "-", "/", ":", ";", "(", ")", "$", "&", "@", "£", "¥", "~", "return"], 
+    ["redo", " ", "...", ".", ",", "?", "!", "'", '"', "_", "€"], 
+    ["selectKeyboard", ".?123", "microphone", "space", ".?123", "scribble", "hideKeyboard"],
```
Comment (andrewtavis) [inline Keyboards/LanguageKeyboards/German/DEInterfaceVariables.swift:69]: See all of these are also experiencing formatting errors :) I can definitely fix this post merge if it's a problem 🙃

**Comment by andrewtavis** [comment]
Hey @nyfz18! Made some comments above :) I'd also be happy to merge and fix these errors myself, and then you can use the German keyboard as a basis for the PRs for the other languages 😊

**Comment by nyfz18** [comment]
sure! In that case, I'll get started with the other keyboards!

**Review by andrewtavis — APPROVED** [review]
As per discussion, approving the current commits and will send along some changes to fix that then @nyfz18 can implement for future PRs 😊

**Comment by andrewtavis** [comment]
Will send along the changes for this shortly and write in the issue, @nyfz18!

File: Keyboards/LanguageKeyboards/German/DEInterfaceVariables.swift (hunk 3 above)
Comment (andrewtavis) [reply Keyboards/LanguageKeyboards/German/DEInterfaceVariables.swift:68]: I screwed up with my suggestion to you on how to include `"`, @nyfz18 :) Sorry for the confusion! It should have been `"\""` 😊

**Pull request merged**